backend/gateway/meta/meta.go

```diff
@@ -135,3 +137,35 @@ func resolvePattern(pb proto.Message, pattern *apiv1.Pattern) *auditv1.Resource
 	}
 	return &auditv1.Resource{TypeUrl: pattern.TypeUrl, Id: resourceName}
 }
+
+// isValidInterface will return true if the type is proto.message and value is not nil
+func isValidInterface(body interface{}) bool {
+	switch v := body.(type) {
+	// type we want is proto.message
+	case proto.Message:
+		// want to use a value that is not nil
+		return v.ProtoReflect().IsValid()
+	default:
+		return false
+	}
+}
+
+// APIBody returns a API request/response interface as an anypb.Any message.
+func APIBody(body interface{}) (*anypb.Any, error) {
+	if !isValidInterface(body) {
```

Comment:
this check is no longer needed afaict since the any marshaller gracefully handles (*myProtoStruct)(nil).

you would `return nil, nil` from the check below instead for same effect.

Reply:
removed this check and instead do
```
	m, ok := body.(proto.Message)
	if !ok {
		// body is not the type/value we want to process
		return nil, nil
	}
```